packages/ember-simple-auth-cookie-store/tests/simple-auth-cookie-store/stores/cookie-test.js

```diff
@@ -51,7 +51,45 @@ describe('Stores.Cookie', function() {
       this.store.cookieDomain = 'example.com';
       this.store.persist({ key: 'value' });
 
-      expect(document.cookie).to.eq('');
+      expect(document.cookie).to.not.contain('test:session=%7B%22key%22%3A%22value%22%7D');
+    });
+  });
+
+  describe('#renew', function() {
+    beforeEach(function() {
+      this.store = Cookie.create();
+      this.store.cookieName = 'test:session';
+      this.store.cookieExpirationTime = 60; // 1 min
+      this.store.expires = new Date().getTime() + this.store.cookieExpirationTime * 1000;
+      this.store.persist({key: this.store.expires});
+    });
+
+    it('session data cookie name should equal "test:session"', function() {
+      expect(this.store.cookieName).to.eq('test:session');
+    });
```

Comment:
no it doesn't sorry. I removed "session data cookie name should equal "test:session"" and changed persist to renew.